recipe-client-addon/lib/NormandyApi.jsm

```diff
@@ -82,19 +82,23 @@ this.NormandyApi = {
     return this.absolutify(url);
   },
 
-  async fetchRecipes(filters = {enabled: true}) {
-    const signedRecipesUrl = await this.getApiUrl("recipe-signed");
-    const recipesResponse = await this.get(signedRecipesUrl, filters);
-    const rawText = await recipesResponse.text();
-    const recipesWithSigs = JSON.parse(rawText);
+  async fetchSignedObjects(type, filters) {
```

Comment:
I think this is not the best level of reuse; having a function that verified objects with signatures, and maybe a shorthand for fetching and parsing JSON from the API, would be a little cleaner (less state to keep in your head when trying to understand the function, etc.). 

I'm haven't yet found a great articulation for that feeling though, so I wouldn't call it a blocker.